src/executables/unload_schema.c

```diff
@@ -636,7 +646,7 @@ get_ordered_classes (print_output & output_ctx, MOP * class_table)
  *    class(in): class MOP
  */
 static void
-emit_class_owner (print_output & output_ctx, MOP class_)
+emit_class_owner (extract_context & ctxt, print_output & output_ctx, MOP class_)
```

Comment:
I found that in this function `extract_context & ctxt` parameter is not used.
Is this for function unity or is there any probability that this parameter will be used?

Reply:
Thank you for your comment.